debezium-server-voyager/debezium-server-voyagerexporter/src/main/java/io/debezium/server/ybexporter/YbExporterConsumer.java

```diff
@@ -191,6 +296,7 @@ private void checkForEndMigrationAndHandle() {
             LOGGER.info("End migration processing complete. Exiting...");
             shutDown = true; // to ensure that no event gets written after switch operation.
         }
+        releaseLockFile();
```

Comment:
same question: wouldn't system.Exit automatically handle it because of the exit hook?